utils/swagger.ts

```diff
@@ -59,6 +59,10 @@ const options: swaggerJsdoc.Options = {
                 name: 'Images',
                 description: 'A set of endpoints related to obtaining placeholder image urls',
             },
+            {
```

Comment:
Can you just confirm this is in the newer place of swagger-config under utils. We do not directly update the swagger docs anymore.

Reply:
Hi, I added it here seeing the tags that are present in the swagger page. So, yes this is where we can add the tags for the swagger page.